src/components/MultiStepForm/Answers.js

```diff
@@ -68,10 +68,15 @@ class Answers extends React.Component {
   }
 
   concernClick = () => global.alert('Let\'s talk about it in messenger, click "get started" if messenger asks you')
+  getMaxWidth = () => {
+    const maxLength = this.props.answers.reduce(
+      (prev, current) => ((prev.text.length > current.text.length) ? prev : current),
+    ).text.replace(/ /g, '').length
+    return maxLength * 12
+  }
```

Comment:
See bug below. Why calculate text and not button outer width directly?

![image](https://user-images.githubusercontent.com/6963212/39124794-6d1cdde4-4705-11e8-8f96-61888c03b953.png)